### ISSUE TITLE
-Wvisibility in include/trace/events/iomap.h

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#644]
Reported-by: kernelci.org bot <bot@kernelci.org>
```
    ./../include/trace/events/iomap.h:52:1: warning: declaration of 'struct inode' will not be visible outside of this function [-Wvisibility]
```

**Comment by nathanchance** [comment]
I have the proper fix for this sitting in my patch stack, I just need to find the time to send this. It comes from `CONFIG_KERNEL_HEADER_TEST` I believe, which automatically compiles headers as standalone units to make sure they are self-contained.

```diff
diff --git a/include/trace/events/iomap.h b/include/trace/events/iomap.h
index 1da90e743e6e..4c29c6901a34 100644
--- a/include/trace/events/iomap.h
+++ b/include/trace/events/iomap.h
@@ -12,6 +12,7 @@
 #if !defined(_TRACE_IOMAP_H) || defined(TRACE_HEADER_MULTI_READ)
 #define _TRACE_IOMAP_H
 
+#include <linux/fs.h>
 #include <linux/tracepoint.h>
 
 DECLARE_EVENT_CLASS(iomap_page_class,
```

**Comment by nathanchance** [comment]
Fixed in `next-20190904`: https://git.kernel.org/next/linux-next/c/32e3cf9f41c70239e2c0172c4f3cebb5c3700cc7
